quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartOptimizedHelp.windows.approved.txt

```diff
@@ -109,6 +113,15 @@ HTTP(S):
                        automatically detected based on the file name. If 'fips-mode' is set to
                        'strict' and no value is set, it defaults to 'BCFKS'.
 
+Config:
+
+--config-keystore <config-keystore>
+                     Specifies a path to the KeyStore Configuration Source.
+--config-keystore-password <config-keystore-password>
+                     Specifies a password to the KeyStore Configuration Source.
+--config-keystore-type <config-keystore-type>
+                     Specifies a type of the KeyStore Configuration Source. Default: PKCS12.
+

```

Comment:
Why these changes were provided? Only the `--https-client-auth` should be changed, right?

Reply:
The windows files are not well maintained, I have a utility to just accept the actual so it will pick up any changes that are still needed. I have filtered those out for a couple of prs, but it makes more sense to just have everything up to date.

Reply:
Ok, works for me. What kind of utility do you use for it?

Reply:
Just a java class that I run from the ide after the test. It would be better to check in something that can run from maven. However the treatment of the windows files isn't completely correct - replacing kc.sh with kc.bat can cause the line length to change and multiple lines need then need to be reformatted.